spec/spec_helper.rb

```diff
@@ -256,10 +264,26 @@ def clean_up_test_environment()
   return
 end
 
-def cleanup_home_and_remote()
+def cleanup_home_and_remote(unified: true)
   FileUtils.rm_rf(FIG_HOME)
-  FileUtils.rm_rf(FIG_REMOTE_DIR)
-
+  #FileUtils.rm_rf(FIG_REMOTE_DIR)
```

Comment:
Are you leaving this here commented out on purpose?

Reply:
No.  I just missed it when I did other similar cleanup.  Now gone.